Error: Cannot find module 'ioredis'

I just deployed an app using the "deploy to heroku" button, and I'm getting this error from the web process:

```
❯ heroku logs -t -a npm-register-zeke
2016-08-22T00:07:38.256658+00:00 app[web.1]: Error: Cannot find module 'ioredis'
2016-08-22T00:07:38.256659+00:00 app[web.1]:     at Function.Module._resolveFilename (module.js:455:15)
2016-08-22T00:07:38.256659+00:00 app[web.1]:     at Function.Module._load (module.js:403:25)
2016-08-22T00:07:38.256661+00:00 app[web.1]:     at require (internal/module.js:20:19)
2016-08-22T00:07:38.256660+00:00 app[web.1]:     at Module.require (module.js:483:17)
2016-08-22T00:07:38.256661+00:00 app[web.1]:     at Object.<anonymous> (/app/lib/redis.js:6:17)
2016-08-22T00:07:38.256662+00:00 app[web.1]:     at Module._compile (module.js:556:32)
2016-08-22T00:07:38.256663+00:00 app[web.1]:     at Object.Module._extensions..js (module.js:565:10)
2016-08-22T00:07:38.256664+00:00 app[web.1]:     at tryModuleLoad (module.js:432:12)
2016-08-22T00:07:38.256663+00:00 app[web.1]:     at Module.load (module.js:473:32)
2016-08-22T00:07:38.256665+00:00 app[web.1]:     at Function.Module._load (module.js:424:3)
2016-08-22T00:07:38.256665+00:00 app[web.1]:     at Module.require (module.js:483:17)
2016-08-22T00:07:38.256666+00:00 app[web.1]:     at Object.<anonymous> (/app/lib/npm.js:7:13)
2016-08-22T00:07:38.256666+00:00 app[web.1]:     at require (internal/module.js:20:19)
2016-08-22T00:07:38.256667+00:00 app[web.1]:     at Module._compile (module.js:556:32)
2016-08-22T00:07:38.256668+00:00 app[web.1]:     at Object.Module._extensions..js (module.js:565:10)
2016-08-22T00:07:38.256668+00:00 app[web.1]:     at Module.load (module.js:473:32)
2016-08-22T00:07:38.265694+00:00 app[web.1]: 
2016-08-22T00:07:38.272683+00:00 app[web.1]: npm ERR! Linux 3.13.0-85-generic
2016-08-22T00:07:38.272857+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-08-22T00:07:38.273101+00:00 app[web.1]: npm ERR! npm  v3.10.3
2016-08-22T00:07:38.272992+00:00 app[web.1]: npm ERR! node v6.4.0
2016-08-22T00:07:38.273219+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2016-08-22T00:07:38.273319+00:00 app[web.1]: npm ERR! npm-register@2.0.2 start: `node server.js`
2016-08-22T00:07:38.273396+00:00 app[web.1]: npm ERR! Exit status 1
2016-08-22T00:07:38.273579+00:00 app[web.1]: npm ERR! Failed at the npm-register@2.0.2 start script 'node server.js'.
2016-08-22T00:07:38.273489+00:00 app[web.1]: npm ERR! 
2016-08-22T00:07:38.273661+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-08-22T00:07:38.273743+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the npm-register package,
2016-08-22T00:07:38.273823+00:00 app[web.1]: npm ERR! not with npm itself.
2016-08-22T00:07:38.273993+00:00 app[web.1]: npm ERR!     node server.js
2016-08-22T00:07:38.274070+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2016-08-22T00:07:38.273912+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2016-08-22T00:07:38.274151+00:00 app[web.1]: npm ERR!     npm bugs npm-register
2016-08-22T00:07:38.274231+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2016-08-22T00:07:38.274308+00:00 app[web.1]: npm ERR!     npm owner ls npm-register
2016-08-22T00:07:38.274388+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
     "fs-extra": "0.30.0",
     "got": "6.3.0",
     "htpasswd-auth": "2.0.0",
+    "ioredis": "^2.3.0",
     "koa": "1.2.1",
     "koa-compress": "1.0.9",
     "koa-router": "5.4.0",

